volunteers.js

```diff
@@ -347,3 +347,16 @@ async function getCurrentMilestone(googleClient) {
 
   return bestMilestone;
 }
+
+async function discordIdToTag(guild, id) {
+    const match = id.match(/<@!?(\d+)>/);
+    if (match !== null) {
+        try {
+            volunteerMember = await guild.fetchMember(match[1]);
+        } catch (err) {
+            console.error("There is no guild member with the provided ID.", err);
```

Comment:
Would it crash the bot when we encounter this, since it looks like we are still trying to access `volunteerMember` when it doesn't exist? Would we need to return `null` here as well?

Reply:
Whoops.... thanks

Reply:
Schweet